updating/index.adoc

```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
 
 * xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[About the OpenShift Update Service]
 * xref:../updating/understanding-the-update-service.adoc#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
 
 [id="updating-clusters-overview-install-and-configure-update-service"]
 == Installing and configuring the OpenShift Update Service
 xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: Clusters with internet accessibility can access public APIs; clusters in a restricted network are unable to access public APIs for update information. To provide a similar upgrade experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
-For more information, see the following actions:
+For more information, see the following procedures:
 
 * xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
 * xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
 * xref:../updating/installing-update-service.adoc#update-service-uninstall[Deleting an OpenShift Update Service application]
 * xref:../updating/installing-update-service.adoc#update-service-delete-service[Uninstalling the OpenShift Update Service Operator]
 
 [id="updating-clusters-overview-upgrade-channels-and-releases"]
-== Upgrade channels and releases
-xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following information:
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version. For more information, see the following information:
```

Comment:
Last sentence: Earlier, something similar was a personal preference. Here, it's more important. We don't want to use "information" twice in a sentence. Use:
"For more information, see the following:"

Reply:
Agree, addressed